doc/nrf/config_and_build/configuring_app/advanced_building.rst

```diff
@@ -0,0 +1,109 @@
+.. _building_advanced:
+
+Advanced building procedures
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can customize the basic :ref:`building <building>` procedures in a variety of ways, depending on the configuration of your project.
+
+.. _gs_modifying_build_types:
+.. _modifying_build_types:
+
+Configuring build types
+***********************
+
+.. build_types_overview_start
+
+When the ``CONF_FILE`` variable contains a single file and this file follows the naming pattern :file:`prj_<buildtype>.conf`, then the build type will be inferred to be *<buildtype>*.
+The build type cannot be set explicitly.
+The *<buildtype>* can be any string, but it is common to use ``release`` and ``debug``.
```

Comment:
We should probably not be documenting this because I'm hoping to dump this feature soon and have it replaced with something universal

Reply:
I would not want to remove it here, as this affects a lot of pages for apps and samples. Let's review the docs when the feature is being replaced. When do you think this might happen?